tests/unit/components/Filter/FilterInput.spec.js

```diff
@@ -598,8 +628,9 @@ describe('FilterInput', () => {
 
       await flushPromises();
 
+      suggestedTags = wrapper.findComponent({ ref: 'suggestedTags' });
       expect(suggestedTags.exists()).toBe(false);
-      expect(wrapper.emitted('show-suggested-tags')).toEqual([[true], [false]]);
+      expect(wrapper.emitted('show-suggested-tags')).toBeTruthy();
```

Comment:
Not sure if we should check truthy for arrays since they could be truthy if they're empty